WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -306,9 +306,19 @@ + (NSDictionary *)processArguments:(XCUIApplication *)app
   [locationManager setPausesLocationUpdatesAutomatically:NO];
   [locationManager startUpdatingLocation];
 
-  CLAuthorizationStatus authStatus = [locationManager respondsToSelector:@selector(authorizationStatus)]
-    ? (CLAuthorizationStatus) [[locationManager performSelector:@selector(authorizationStatus)] integerValue]
-    : [CLLocationManager authorizationStatus];
+
+  CLAuthorizationStatus authStatus;
+  if ([locationManager respondsToSelector:@selector(authorizationStatus)]) {
+    // To fix warning
```

Comment:
this comment might be confusing for people ho don't know the context. My proposal is to remove it